bssl-compat/openssl.cmake

```
@@ -20,14 +42,17 @@ ExternalProject_Add(
         TEST_COMMAND ""
         INSTALL_COMMAND make install
         INSTALL_DIR ${OPENSSL_INSTALL_DIR}
-)
+    )
+
```

```suggestion
    file(MAKE_DIRECTORY ${OPENSSL_INCLUDE_DIR})
```
The $OPENSSL_INCLUDE_DIR doesn't exist until the first build is run, so I had to add this to get it working

---

That was a good fix.